src/minter-did/src/id256.rs

```diff
@@ -175,6 +218,23 @@ impl Storable for Id256 {
     }
 }
 
+#[cfg(feature = "runes")]
+impl From<RuneId> for Id256 {
```

Comment:
To avoid multiple `#[cfg(feature = "runes")]` here and in the tests, please move all code of this feature in a `runes.rs` file and feature gate it in the parent mod. i.e.:
```rust
// in the runes.rs parent mod:
#[cfg(feature = "runes")]
pub use runes;
```

Reply:
Done